sal_interfaces/HVAC/HVAC_Events.xml

```diff
@@ -1,23 +1,92 @@
-  <Enumeration>CrackId_crack01=1,CrackId_crack02=2</Enumeration>
-  <Enumeration>FancoilId_fancoil01=1,FancoilId_fancoil02=2,FancoilId_fancoil03=3,FancoilId_fancoil04=4,FancoilId_fancoil05=5,FancoilId_fancoil06=6,FancoilId_fancoil07=7,FancoilId_fancoil08=8,FancoilId_fancoil09=9,FancoilId_fancoil10=10,FancoilId_fancoil11=11,FancoilId_fancoil12=12</Enumeration>
-  <Enumeration>ManejadoraId_manejadoraSblanca=1,ManejadoraId_manejadoraSlimpia=2</Enumeration>
-  <Enumeration>ManejadoraLowerId_manejadoraLower01=1,ManejadoraLowerId_manejadoraLower02=2,ManejadoraLowerId_manejadoraLower03=3,ManejadoraLowerId_manejadoraLower04=4</Enumeration>
-  <SALEvent>
-    <Subsystem>HVAC</Subsystem>
-    <EFDB_Topic>HVAC_logevent_detailedState</EFDB_Topic>
-    <Description>Event for the detailedState.</Description>
-    <item>
-      <EFDB_Name>detailedState</EFDB_Name>
-      <Description>Detailed state</Description>
-      <IDL_Type>long</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
-  </SALEvent>
+  <Enumeration>
+    SummaryState_DisabledState,
+    SummaryState_EnabledState,
+    SummaryState_FaultState,
+    SummaryState_OfflineState,
+    SummaryState_StandbyState
+  </Enumeration>
+  <Enumeration>
+    DisabledState_DisabledState,
+    DisabledState_EnabledState,
+    DisabledState_FaultState,
+    DisabledState_OfflineState,
+    DisabledState_StandbyState
+  </Enumeration>
+  <Enumeration>
+    HvacTopicId_chiller01P01=0X0001,
+    HvacTopicId_chiller02P01=0X0002,
+    HvacTopicId_chiller03P01=0X0003,
+    HvacTopicId_crack01P02=0X0004,
+    HvacTopicId_crack02P02=0X0005,
+    HvacTopicId_fancoil01P02=0X0006,
+    HvacTopicId_fancoil02P02=0X0007,
+    HvacTopicId_fancoil03P02=0X0008,
+    HvacTopicId_fancoil04P02=0X0009,
+    HvacTopicId_fancoil05P02=0X000A,
+    HvacTopicId_fancoil06P02=0X000B,
+    HvacTopicId_fancoil07P02=0X000C,
+    HvacTopicId_fancoil08P02=0X000D,
+    HvacTopicId_fancoil09P02=0X000E,
+    HvacTopicId_fancoil10P02=0X000F,
+    HvacTopicId_fancoil11P02=0X0010,
+    HvacTopicId_fancoil12P02=0X0011,
+    HvacTopicId_manejadoraLower01P05=0X0012,
+    HvacTopicId_manejadoraLower02P05=0X0013,
+    HvacTopicId_manejadoraLower03P05=0X0014,
+    HvacTopicId_manejadoraLower04P05=0X0015,
+    HvacTopicId_manejadoraSblancaP04=0X0016,
+    HvacTopicId_manejadoraSlimpiaP04=0X0017,
+    HvacTopicId_vea01P01=0X0018,
+    HvacTopicId_vea01P05=0X0019,
+    HvacTopicId_vea08P05=0X001A,
+    HvacTopicId_vea09P05=0X001B,
+    HvacTopicId_vea10P05=0X001C,
+    HvacTopicId_vea11P05=0X001D,
+    HvacTopicId_vea12P05=0X001E,
+    HvacTopicId_vea13P05=0X001F,
+    HvacTopicId_vea14P05=0X0020,
+    HvacTopicId_vea15P05=0X0021,
+    HvacTopicId_vea16P05=0X0022,
+    HvacTopicId_vea17P05=0X0023,
+    HvacTopicId_vec01P01=0X0024,
+    HvacTopicId_vin01P01=0X0025,
+    HvacTopicId_vex03LowerP04=0X0026,
+    HvacTopicId_vex04CargaP04=0X0027
```

Comment:
Please consider removing the Hvac prefix because it does not add much, if any, information.

Perhaps DeviceId instead? "Topic" seems an odd name for something that you enable and configure. But if that's the terminology the vendor uses, "TopicId" would do.

Reply:
I agree and will make the changes. "DeviceId" is fine. I can make the translation in the CSC.